content/cs/logs.md

```diff
@@ -0,0 +1,16 @@
+## XI. Logy
+### S logy zacházejte jako s proudy událostí.
+
+*Logy* poskytují náhled na chování běžící aplikace. V prostředí serveru se obvykle zapisují do souboru na disku ("logfile"), ale to je jen výstupní formát.
+
+Logy jsou [proudy](https://adam.herokuapp.com/past/2011/4/1/logs_are_streams_not_files/) agregovaných a časově seřazených událostí, posbíraných z výstupních proudů všech běžících procesů a podpůrných služeb. Logy jsou ve své syrové formě typicky v textovém formátu s jednou událostí na řádek (avšak výpis vyjímky může zabírat i více řádků). Logy nemají žádný pevný začátek ani konec, ale plynule proudí po celou dobu běhu aplikace.
+
+**Twelve-factor aplikace se nikdy nestará o routování anebo ukládání svého výstupního proudu.** Neměla by se ani pokoušet zapisovat nebo si spravovat své log soubory. Místo toho zapisuje každy její běžící proces nebufferovaně na `stdout`. Během lokálního vývoje vidí vývojář proud událostí ve svém terminále a sleduje chování aplikace.
```

Comment:
Wrong ending: "ve svém terminále" => "ve svém terminálu"

Reply:
to je podle vzoru motýle: https://www.meriva-preklady.cz/clanek/motyle-motyly-andrej-babis-sklonovani-pravopis_125